src/pages/Auth/SignupPage/Steps/VerificationStep/index.tsx

```diff
@@ -191,60 +159,71 @@ function Verification({ onNext, onBack, setUserType }: VerificationProps) {
-                  )}
-                  {phoneMessage?.type === 'error' && phoneMessage.code === 'ALREADY_REGISTERED' && (
-                  <>
-                    <button
-                      onClick={goToLogin}
-                      type="button"
-                      className={styles['label-link-button']}
-                    >
-                      로그인하기
-                    </button>
-                    <span className={styles['label-link-split']}>|</span>
-                    <a
-                      href="https://open.kakao.com/o/sgiYx4Qg"
-                      className={styles['label-link-wrapper__button']}
-                    >
-                      문의하기
-                    </a>
-                  </>
-                  )}
-                </PCCustomInput>
-                <button
-                  type="button"
-                  onClick={() => checkPhoneNumber(phoneNumber)}
-                  className={styles['check-button']}
-                  disabled={!/^01[016789][0-9]{7,8}$/.test(field.value)}
-                >
-                  {buttonText}
-                </button>
+              <div>
+                <div className={styles['input-with-button']}>
+                  <PCCustomInput
+                    htmlFor="phone_number"
+                    labelName="휴대전화"
+                    {...field}
+                    onChange={(e) => {
+                      field.onChange(e);
+                      setPhoneMessage(null);
+                      stopTimer();
+                      setValue('isCorrect', false);
+                      setVerificationMessage(null);
+                      setValue('verification_code', '');
+                    }}
+                    placeholder="숫자만 입력해 주세요."
+                    isRequired
+                    message={phoneMessage}
+                    disabled={isDisabled || isVerified}
+                    isDelete={!isVerified}
+                    onClear={() => {
+                      setPhoneMessage(null);
+                      setButtonText('인증번호 발송');
+                    }}
+                  >
+                    {phoneMessage?.type === 'success' && (
+                    <div className={styles['label-count-number']}>
+                      {' '}
+                      남은 횟수 (
+                      {smsSendCount}
+                      /5)
+                    </div>
+                    )}
+                    {phoneMessage?.type === 'error' && phoneMessage.code === 'ALREADY_REGISTERED' && (
+                    <>
+                      <button
+                        onClick={goToLogin}
+                        type="button"
+                        className={styles['label-link-button']}
+                      >
+                        로그인하기
+                      </button>
+                      <span className={styles['label-link-split']}>|</span>
+                      <a
+                        href="https://open.kakao.com/o/sgiYx4Qg"
```

Comment:
나중에 상수로 빼도 좋을 거 같네요!
```suggestion
export const INQUIRY_URL = "https://open.kakao.com/o/sgiYx4Qg"
                        href={INQUIRY_URL}
```

Reply:
상수화 시켰습니다 감사합니다! 😃